src/utils/ucx/ucx_utils.cpp

```diff
@@ -468,12 +466,12 @@ std::string nixlUcxWorker::epAddr()
         NIXL_WARN << "Unable to query UCX endpoint address";
         return {};
     }
-    const std::string result = nixlSerDes::_bytesToString(wattr.address, wattr.address_length);
+    const auto result = std::string(reinterpret_cast<const char*>(wattr.address), wattr.address_length);  // Deserialize?
```

Comment:
Not needed. The worker address was obtained using the UCP API of UCX library. So it was not serialized by NIXL.